src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/PrisonerSearchClientIntTest.kt

```diff
@@ -0,0 +1,80 @@
+package uk.gov.justice.digital.hmpps.personrecord.client
+
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.equalTo
+import com.github.tomakehurst.wiremock.client.WireMock.post
+import com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.personrecord.PrisonerSearchClient
+import uk.gov.justice.digital.hmpps.personrecord.client.model.PossibleMatchCriteria
+import uk.gov.justice.digital.hmpps.personrecord.integration.IntegrationTestBase
+import java.time.LocalDate
+
+class PrisonerSearchClientIntTest : IntegrationTestBase() {
+
+  @Autowired
+  lateinit var restClient: PrisonerSearchClient
+
+//  @BeforeEach
+//  fun setup() {
+//    wireMockExtension
+//    wireMockServer.start()
+//    configureFor("localhost", 8090)
+//  }
+//
+//  @AfterEach
+//  fun tearDown() {
+//    wireMockServer.stop()
+//  }
+
```

Comment:
Looks like we don't need this commented out code

Reply:
Agree.